packages/js/uri-resolvers/src/__tests__/retry-resolver.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { Uri, UriResolutionContext } from "@polywrap/core-js";
+import { expectHistory } from "./util";
+import { RecursiveResolver, RetryResolver, RetryResolverOptions } from "../helpers";
+import { ClientConfigBuilder, PolywrapClient } from "@polywrap/client-js";
+import { ExtendableUriResolver } from "@polywrap/uri-resolver-extensions-js";
+import { PackageToWrapperCacheResolver, WrapperCache } from "../cache";
+import { StaticResolver } from "../static";
+import {
+  embeds,
+  ipfsProviders,
+  plugins,
+} from "@polywrap/client-config-builder-js/build/bundles/default";
+
+jest.setTimeout(200000);
+
+const getClientWithRetryResolver = (retryOptions: RetryResolverOptions): PolywrapClient => {
+  const ipfsResolverUri = Uri.from("wrap://package/ipfs-resolver");
+  const resolver = RecursiveResolver.from(
+    PackageToWrapperCacheResolver.from(
```

Comment:
We don't need all these resolvers just to test the RetryResolver.
A simple test resolver instead should do the trick.
Checkout this for inspiration: https://github.com/polywrap/toolchain/pull/1586/files#diff-57a872ca435da8f15bb68754ef07491fa13b9b2d043ffed209e474f495c7b662